plugins/operators/mattermost_operator.py

```diff
@@ -0,0 +1,30 @@
+from airflow.utils.decorators import apply_defaults
+import requests
+
+class MattermostWebhookOperator():
```

Comment:
According to the [docs](https://airflow.apache.org/docs/apache-airflow/1.10.15/howto/custom-operator.html) operators must be descendands of `BaseOperator`. There's two options here:
- `BaseOperator` is the parent class. This is the simplest approach.
-`SimpleHTTPOperator` is the parent class. This looks like a better approach, as all HTTP handling can be deferred to `SimpleHTTPOperator` methods. This internally uses `HttpHook` which handles HTTP failures, retries etc.